Global/JetBrains.gitignore

```diff
@@ -7,6 +7,13 @@
 .idea/**/dictionaries
 .idea/**/shelf
 
+# Rider IDE-specific stuff
+# Reference: https://rider-support.jetbrains.com/hc/en-us/articles/207097529
+.idea/**/modules.xml
+.idea/**/contentModel.xml
+*.iml
```

Comment:
What about `.idea/**/*.iml` instead? Is there any iml file generated outside of .idea folder?

Reply:
I'm 👎 to this rule because `*.iml` files are considered share-able in the reference link on line 2:

> This format is used by all the recent IDE versions by default. Here is what you need to share:
>
> - All the files under .idea directory in the project root except the **workspace.xml**, **usage.statistics.xml**, and **tasks.xml** files which store user specific settings
> - All the **.iml** module files that can be located in different module directories (applies to IntelliJ IDEA)